### PR TITLE
Fix AppTheme tests

### PR DESCRIPTION
**Pull request by EmmanuelMess**
#### Automatic tests

- [ ] Added test cases
  
#### Manual tests
- [ ] Done  
  

#### Build tasks success  

Successfully running following tasks on local:
- [x] `./gradlew assembledebug`
- [x] `./gradlew spotlessCheck`

### CODE REVIEW

**Comment by TranceLove** [comment]
I got a compile error at `Billing.java` complaining missing `Context` parameter... is it just me?

**Comment by VishalNehra** [comment]
@TranceLove Will try

**Comment by EmmanuelMess** [comment]
> I got a compile error at `Billing.java` complaining missing `Context` parameter... is it just me?

This compiles for me and on CI? Could it be a build config issue?

**Comment by TranceLove** [comment]
> > I got a compile error at `Billing.java` complaining missing `Context` parameter... is it just me?
> 
> This compiles for me and on CI? Could it be a build config issue?

This is weird. I was doing `./gradlew clean spotlessApply spotlessCheck jacocoTestPlayDebugUnitTestReport assembleDebug --stacktrace` from the command line as well...

And even I fixed `AppTheme` there were some failing tests with `TextEditorActivity` and `WriteTextFileCallable`... _why did I put a `\n` there?_

File: app/src/test/java/com/amaze/filemanager/ui/theme/AppThemeTest.kt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        Assert.assertEquals(Theme.DARK, getMaterialDialogTheme(AppTheme.BLACK_INDEX))
+    }
+
+    /**
+     * Test [AppTheme.getSimpleTheme] with mask
+     */
+    @Test
+    fun testSimpleTheme() {
+        val context = ApplicationProvider.getApplicationContext<Context>()
+        val mask = context.resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK == Configuration.UI_MODE_NIGHT_YES
```
[the remark this answers is not in the document]
Comment (EmmanuelMess) [reply app/src/test/java/com/amaze/filemanager/ui/theme/AppThemeTest.kt]: Fixed.

**Comment by EmmanuelMess** [comment]
This makes tests work again for the CI in PRs to 3.7.

**Review by VishalNehra — APPROVED** [review]

**Pull request merged**